### PR TITLE
[F] start tests using mocked API response

### PR DESCRIPTION
**Pull request by DrWilliamsSteven**

### CODE REVIEW

**Comment by AlexanderBrinkmann** [comment]
@DrWilliamsSteven Please check the updated tests.

File: src/controllerSynonyms.spec.ts [hunk 1]
```diff
@@ -91,7 +91,15 @@ test('Testing controller', (t: Test) => {
...
     const _postgres: any = { query: stub().returns(Promise.resolve()) };
 
-    await populateSynonyms({ _postgres })
+    const mockResult = [ [ 'A1BG-AS1', 'FLJ23569' ],
+        [ 'A1CF', 'ACF', 'ASP', 'ACF64', 'ACF65', 'APOBEC1CF' ],
+        [ 'A2M', 'FWP007', 'S863-7', 'CPAMD5' ],
+        [ 'ABAT', 'GABAT' ],
+        [ 'ABCA1', 'TGD' ] ];
+
+    const _allSynonyms: any = stub().returns(Promise.resolve(mockResult));
```
[the remark this answers is not in the document]
Comment (DrWilliamsSteven) [reply src/controllerSynonyms.spec.ts]: Also, in your test above `'Test getSynonyms function when term is not in database and not available by HUGO'` you should inject and mock `updateSynonyms`

[the remark this answers is not in the document]
Comment (AlexanderBrinkmann) [reply src/controllerSynonyms.spec.ts]: Done

**Comment by AlexanderBrinkmann** [comment]
@DrWilliamsSteven @Istar-Eldritch @mareq 
Please review the code I pushed and give me a short feedback!

**Comment by DrWilliamsSteven** [comment]
👍 🚢 it!

**Pull request merged**